metaflow/plugins/cards/ui/src/components/line-chart.svelte

```diff
@@ -21,32 +21,44 @@
   );
 
   export let componentData: types.LineChartComponent;
-  $: ({ config, data, labels } = componentData);
-
   let el: HTMLCanvasElement;
+  let chart: Chart;
 
-  const chartConfiguration: ChartConfiguration = config || {
-    type: "line",
-    data: {
-      labels,
-      datasets: [
-        {
-          backgroundColor: COLORS_LIST[2],
-          borderColor: COLORS_LIST[2],
-          data: data || [],
-        },
-      ],
-    },
-    options: {
-      plugins: {
-        legend: {
-          display: false,
+  const createChart = () => {
+    const { config, data, labels } = componentData;
```

Comment:
Is this better than leaving it reactive above? Since the json loading is async, i'm not entirely sure this is ideal.

Reply:
When i tried runnning it without this code change then the charts were not getting updated. Didn't know why so I tried something like this.

Reply:
In lines that follow, there is a `$` function handling some updates  (Is that incorrectly done ?) I don't know the exact syntax.

Reply:
if it works, i would leave as is! 

the next version of svelte will solve for this problem entirely anyway.